Wiki fails

### Container image name

rocker/R4.5.2

### Container image digest

_No response_

### What operating system are you seeing the problem on?

Linux

### System information

_No response_

### Bug description

The  build\reports\wiki_home.Rmd fails  when called in the action reports.yml

-----

The error in github looks like 
processing file: wiki_home.Rmd
1/14                    
2/14 [setup]            
3/14                    
4/14 [data_manipulation]
Error in `list_unchop()`:
! Can't merge the outer name `bakefiles/4.4.3.docker-bake.json` with a vector of length > 1.
Please supply a `.name_spec` specification.
Backtrace:
     ▆
  1. ├─dplyr::mutate(...)
  2. ├─purrr::list_rbind(...)
  3. │ └─purrr:::check_list_of_data_frames(x)
  4. │   └─vctrs::obj_check_list(x, call = error_call)
  5. ├─purrr::map(...)
  6. │ └─purrr:::map_("list", .x, .f, ..., .progress = .progress)
  7. │   └─purrr:::vctrs_vec_compat(.x, .purrr_user_env)
  8. ├─dplyr::pull(...)
  9. ├─dplyr::arrange(...)
 10. ├─dplyr:::arrange.data.frame(...)
 11. │ └─dplyr:::arrange_rows(.data, dots = dots, locale = .locale)
 12. │   └─vctrs:::vec_order_radix(...)
 13. │     └─vctrs (local) `<fn>`()
 14. │       └─vctrs::vec_proxy_equal(x = x)
 15. │         └─vctrs:::proxy_equal_numeric_version(x)
 16. │           └─vctrs::vec_interleave(!!!x, .ptype = integer())
 17. │             └─vctrs::list_unchop(...)
 18. └─rlang::abort(message = message)

Quitting from wiki_home.Rmd:28-102 [data_manipulation]
Execution halted
make: *** [Makefile:81: wiki-home] Error 1

----

I think this is to do with line 59, or maybe the input json file into that section of code. 





### How to reproduce this bug?

```sh

```

--- a/build/reports/wiki_home.Rmd
+++ b/build/reports/wiki_home.Rmd
@@ -48,6 +48,7 @@ df_definitions <- fs::dir_ls(
   path = "bakefiles",
   regexp = r"(.+\.docker-bake.json$)"
 ) |>
+  as.character() |>
   tibble::tibble(file = _) |>
   dplyr::arrange(
     stringr::str_extract(file, r"(\d+\.\d+\.\d+)") |>

